src/bindings/c/include/c_api/ie_c_api.h

```diff
@@ -24,6 +24,7 @@
 
 #include <stdint.h>
 #include <stdio.h>
+#include "va/va.h"
```

Comment:
what if I compile for Windows? It does not have such header file

Reply:
Can I include va.h like this [remote_context.hpp#L21](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gpu/include/intel_gpu/plugin/remote_context.hpp#L21)?
```suggestion
#ifdef _WIN32
// something else to include
#else
# include "va/va.h"
#endif
```
Or can I use this [va.h](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gpu/include/va/va.h) file